pkg/integration/tests/commit/staged.go

```diff
@@ -53,8 +53,11 @@ var Staged = NewIntegrationTest(NewIntegrationTestArgs{
 				Contains(commitMessage),
 			)
 
-		t.Views().StagingSecondary().IsFocused()
-
-		// TODO: assert that the staging panel has been refreshed (it currently does not get correctly refreshed)
+		t.Views().Staging().IsFocused().Tap(func() {
```

Comment:
The Tap function is only really needed when we want to make an assertion about another view and then afterwards chain more method calls onto the original one. But in this case it's not necessary; we can instead go:
```go
t.Views().StagingSecondary().
	IsEmpty()
	
t.Views().Staging().
	IsFocused().
	Content(Contains("+myfile content")).
	Content(DoesNotContain("+with a second line"))

```